CopilotKit/packages/runtime/src/lib/runtime/copilot-runtime.ts

```
@@ -937,9 +963,51 @@ please use an LLM adapter instead.`,
-      throw new Error(`Unknown endpoint type: ${(agentWithEndpoint.endpoint as any).type}`);
+      const { messages, ...stateWithoutMessages } = state;
+      const copilotkitMessages = langchainMessagesToCopilotKit(messages);
+      return {
+        threadId: threadId || "",
+        threadExists: true,
+        state: JSON.stringify(stateWithoutMessages),
+        messages: JSON.stringify(copilotkitMessages),
+      };
     }
+
+    throw new Error(`Agent: ${agent.name} could not be resolved`);
```

Biome (1.9.4)

[error] This code is unreachable

... because either this statement will throw an exception, ...

... this statement will throw an exception, ...

... this statement will throw an exception, ...

... this statement will return from the function, ...

... this statement will throw an exception, ...

... this statement will throw an exception, ...

... this statement will throw an exception, ...

... this statement will return from the function, ...

... or this statement will return from the function beforehand


(lint/correctness/noUnreachable)